bsconfig.json

```diff
@@ -1,26 +1,47 @@
 {
```

Comment:
No big changes, just took the config from here (https://github.com/reasonml-community/reason-apollo-client/tree/master/EXAMPLES), that is why it looks so different.